t/porting/known_pod_issues.dat

```diff
@@ -389,6 +389,7 @@ pod/perlandroid.pod	Verbatim line length including indents exceeds 78 by	3
 pod/perlbook.pod	Verbatim line length including indents exceeds 78 by	1
 pod/perldebguts.pod	Verbatim line length including indents exceeds 78 by	24
 pod/perldebtut.pod	Verbatim line length including indents exceeds 78 by	2
+pod/perldiag.pod	Apparent broken link	2
```

Comment:
There's something wrong here that should be fixed.

Reply:
Fixed now.